mobile/lib/modules/settings/ui/cleanup_device_settings/manual_device_cleanup.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:immich_mobile/modules/backup/models/backup_state.model.dart';
+import 'package:immich_mobile/modules/backup/providers/backup.provider.dart';
+import 'package:immich_mobile/modules/home/ui/delete_diaglog.dart';
+import 'package:immich_mobile/shared/providers/asset.provider.dart';
+
+class ManualDeviceCleanupWidget extends HookConsumerWidget {
+  const ManualDeviceCleanupWidget({
+    Key? key,
+    required this.backupState,
+  }) : super(key: key);
+
+  final BackUpState backupState;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final isUploaded = useState(
+      backupState.allUniqueAssets.length ==
+              backupState.selectedAlbumsBackupAssetsIds.length &&
+          backupState.allUniqueAssets.isNotEmpty,
+    );
+    return Padding(
+      padding: const EdgeInsets.fromLTRB(0, 5, 20, 0),
+      child: Row(
+        children: [
+          SizedBox(
+            width: 250,
+            child: ListTile(
+              title: Text(
+                isUploaded.value
```

Comment:
It seems like there is something wrong with the logic here, my upload process is done - althought not all assets are uploaded fromt the selected album, but I expect to be able to perform the deleting action on the uploaded asset.